src/test/java/io/neonbee/test/helper/DummyVerticleHelper.java

```diff
@@ -249,6 +251,10 @@ public Future<Set<FullQualifiedName>> entityTypeNames() {
                     return succeededFuture(Set.of(fqn));
                 }
             };
+
+            EntityVerticle spy = spy(ev);
+            when(spy.getName()).thenReturn(String.format("_%s-%d", fqn, ev.getClass().getName().hashCode()));
```

Comment:
I'm not sure if FQN is a real improvement. In case I want to deploy 2 different DummyEntityVerticles that provide the same Entity, you would end up in the same situation. It's better than now, because now you can only deploy ONE EntityVerticle, but no real solution, isn't it?

Reply:
I have updated the code to use the object identity hash code instead of the class name hash code.